main.go

```diff
@@ -32,10 +32,9 @@ const (
 )
 
 var (
-	packagesBasePath string
```

Comment:
Seems like we don't use this anymore? My initial intention with this was to be able to pass it as a flag to to the docker container on which directories it should read.

Reply:
I the next step I have a plan to support multiple sources of packages (exactly for the docker environment). I think we can leave it as is.